### PR TITLE
[DO NOT MERGE] Fix RC from FMU for fmu-v5

### PR DESCRIPTION
**Pull request by DanielePettenuzzo**
FMU USART6 for RC should be inverted and in wire mode. Removed internal pull-up on this pin because it was creating weird waveforms on the line. My SBUS receiver is still not detected but at least now the waveforms seem to be clean although some oscillations are present on the power supply. 
SBUS with internal pull-up
sbus_with_pullup
SBUS without internal pull-up (zoom in)
sbus_no_pullup
SBUS without internal pull-up (zoom out)
sbus_no_pull_zoom_out

### CODE REVIEW

File: platforms/nuttx/nuttx-configs/px4fmu-v5/include/board.h [hunk 1]
```diff
@@ -339,7 +339,7 @@
 #define GPIO_UART4_TX    GPIO_UART4_TX_4    /* PD1 */
 
 #define GPIO_USART6_RX   GPIO_USART6_RX_2   /* PG9  */
-#define GPIO_USART6_TX   GPIO_USART6_TX_2   /* PG14 */
+#define GPIO_USART6_TX   (GPIO_ALT|GPIO_AF8|GPIO_SPEED_100MHz|GPIO_PUSHPULL|GPIO_PORTG|GPIO_PIN14)   /* PG14 */
```
Comment (davids5) [inline platforms/nuttx/nuttx-configs/px4fmu-v5/include/board.h]: This is the definition of  GPIO_USART6_TX_2

Comment (DanielePettenuzzo) [reply platforms/nuttx/nuttx-configs/px4fmu-v5/include/board.h]: yes this was just for testing what was going on on the wire.. it is the same definition but I removed an internal pull_up that was distorting the waveform like in the first image above.

**Comment by davids5** [comment]
@DanielePettenuzzo would you try https://github.com/PX4/Firmware/pull/9898 on you mini and see if the sbus works now.

**Comment by davids5** [comment]
Closed via https://github.com/PX4/Firmware/pull/9978 and friends.

**Pull request closed without merging**